app/views/errors/400.html.erb

```diff
@@ -0,0 +1,21 @@
+<% content_for :title, '400 - Bad Request' %>
+
+<section>
```

Comment:
This looks like a partial with the locals: `error_title`, `status_code`, `quote`, `author`. The partial would be useful in case we'll have to make future changes to all of the views. That said, I can see the argument this being simple enough that it's not needed.